analyzer/src/main/resources/UserAgents/TV.yaml

```diff
@@ -2449,3 +2459,162 @@ config:
+#  "Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36 CrKey/1.0.999999 VIZIO SmartCast(Conjure/MTKE-4.6.432.1 FW/3.0.11.3-1 Model/D32h-J09)"
+#  "Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36 CrKey/1.0.999999 VIZIO SmartCast(Conjure/MTKA-5.1.542.1 FW/4.70.27-3 Model/D43fx-F4)"
+#  "Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36 CrKey/1.0.999999 VIZIO SmartCast(Conjure/MTKB-5.1.531.1 FW/2.70.33-1 Model/D50x-G9)"
+#  "Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36 CrKey/1.0.999999 VIZIO SmartCast(Conjure/MTKC-4.6.448.1 FW/1.20.18.13-1 Model/V655-J09)"
+#  "Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36 CrKey/1.0.999999 VIZIO SmartCast(Conjure/MTKC-4.6.448.1 FW/1.20.18.13-1 Model/V555-H1)"
+#  "Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36 CrKey/1.0.999999 VIZIO SmartCast(Conjure/MTKB-5.1.531.1 FW/2.70.33-1 Model/V555-G1)"
+#  "Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.112 Safari/537.36 OPR/36.0.2128.0 OMI/4.8.0.129.ALISHAN6.19 VIZIO-DTV/V7.20.8 (Vizio, D32f-E1, wireless)"
+#  "Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.112 Safari/537.36 OPR/36.0.2128.0 OMI/4.8.0.129.ALISHAN6.19 VIZIO-DTV/V7.20.8 (Vizio, D50f-E1, wireless)"
+#  "Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.112 Safari/537.36 OPR/36.0.2128.0 OMI/4.8.0.129.ALISHAN6.19 VIZIO-DTV/V7.20.8 (Vizio, D48f-E0, wireless)"
+#  "Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.112 Safari/537.36 OPR/36.0.2128.0 OMI/4.8.0.129.ALISHAN6.19 VIZIO-DTV/V7.20.8 (Vizio, D43f-E2, wireless)"
+#  "Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.112 Safari/537.36 OPR/36.0.2128.0 OMI/4.8.0.129.ALISHAN6.19 VIZIO-DTV/V7.20.8 (Vizio, D32f-E1, wired)"
+
+##### Comcast Cable Boxes
+
+# Comcast Set Top Box. Only two of these UAs currently detected but they produce a steady 5M hourly opportunities. Looks like
+# older UA: `Mozilla/5.0 (Linux) AppleWebKit/601.0 (KHTML, like Gecko) Version/8.0 Safari/601.0 WPE` has been almost
+# entirely replaced by UA used in test (as of 2022-02-04). This makes sense since comcast can update all of their
+# boxes at will so we'd likely only ever see 1 variation of this UA at a time.
+- matcher:
+    require:
+      # The detected UAs end with 'WPE' but I don't know how to specify that best in this tool so right now we are
+      # casting a wider net than necessary.
+      - 'agent~"WPE"'
```

Comment:
This should be `endsWith` rather than `contains` but regular regex didn't appear to work here

Reply:
The "ends with" operator looks like this: `agent.product.name}"z"`
https://yauaa.basjes.nl/developer/makingnewrules/#walking-around-the-tree

Reply:
Bah, I forgot about that page, apologies

Reply:
I did some googleing on this term and found this PDF https://iabtechlab.com/wp-content/uploads/2019/12/OTT_CTV_User_Agent_Preliminary_Guidelines_IABTechLab_2019-12.pdf which states:
```
4. User agents lacking device identifiers
Examples:
...
· The Comcast X1 Box returns the default WebPlatformEmbedded (WPE) user agent
“Mozilla/5.0 (Linux; x86_64 GNU/Linux) AppleWebKit/601.1 (KHTML, like Gecko) Version/8.0
Safari/601.1 WPE”
```

I was however unable to find any other reference to this `WebPlatformEmbedded (WPE)`.
Do you have more information on this?

Reply:
I think I found it https://wpewebkit.org/
So I agree with the "Set top box" part. 
I'm not sure about the "Comcast" conclusion: Others may use the same default useragent.

Reply:
That's a valid point, I did over specify the agent

That document was one of the only ones referencing that. Funnily enough ExoPlayerLib/ is next on my list
```
4. User agents lacking device identifiers
Examples:
· The exoplayer library returns the UA "<app name>/<app version> (Linux;Android
<android version>) ExoPlayerLib/<exoplayer version>”
· The Samsung Smart-TV / Tizen devices return “samsung-agent/1.1”
· The Comcast X1 Box returns the default WebPlatformEmbedded (WPE) user agent
“Mozilla/5.0 (Linux; x86_64 GNU/Linux) AppleWebKit/601.1 (KHTML, like Gecko) Version/8.0
Safari/601.1 WPE
```

Reply:
https://webkit.org/wpe/